.github/workflows/test-and-release-svm-contract.yml

```diff
@@ -1,30 +1,41 @@
-name: Release SVM Program
+name: Test and release SVM contracts
 
 on:
+  pull_request:
+    paths:
+      - contracts/svm/**
   push:
     branches:
       - main
     paths:
       - contracts/svm/**
 
+env:
+  CARGO_TERM_COLOR: always
```

Comment:
what does this do out of curiosity?

Reply:
I don't know but it was in test-svm-contract.yml. I just copied it over 😄